lyngk.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Lyngk Website</title>
+
+  <link rel="stylesheet" href="./style.css">
+
+</head>
+
+<body>
+  <html>
+	<head>
+	    <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+		<title>Lyngk Board</title>
+	</head>
+
+	<body>
+	    <div class="container">
+	        <div>
+	            <h1>Lyman's Lyngk Game</h1>
+	        </div>
+            <div id="boardset">
+                <canvas width=400 height=400 id="lyngkboard" ondrop="drop(event)" ondragover="allowDrop(event)" z-axis=-20>
+                Your browser does not support the HTML5
+            </canvas>
+            </div>
+        </div> <!-- .container -->
+	</body>
+</html>
+
+    <script src="./lyngk.js"></script>
+    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>

```

Comment:
Again, unless you intend to use jquery, and/or bootstrap, these do not belong here.